File: C#/BluffinMuffin.Protocol/Lobby/RegisteredMode/CreateUserResponse.cs
namespace BluffinMuffin.Protocol.Lobby.RegisteredMode
{
    /// <summary>
    /// 
    /// </summary>
    public class CreateUserResponse : AbstractBluffinReponse<CreateUserCommand>
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public CreateUserResponse(CreateUserCommand command)
            : base(command)
        {
        }
    }
}

File: Python/bluffin_muffin/protocol/lobby/registered_mode/create_user_response.py
from Python.bluffin_muffin.protocol.abstract_response import AbstractResponse
from create_user_command import CreateUserCommand

__author__ = 'ericmas001@gmail.com'



class CreateUserResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__(obj, CreateUserCommand(obj['Command']))